tests/compiler/test_plugins.py

```diff
@@ -431,14 +436,53 @@ def test_context_and_defaults(snippetcompiler: "SnippetCompilationTest") -> None
     compiler.do_compile()
 
 
-def test_native_types(snippetcompiler: "SnippetCompilationTest") -> None:
+def test_inferred_signatures_logging(snippetcompiler: "SnippetCompilationTest", caplog) -> None:
```

Comment:
@wouterdb I'm adding you as a reviewer to review this test case, i.e. how / what we'll be showing to the user.